aptos-move/aptos-workspace-server/src/main.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Aptos Foundation
```

Comment:
This is the initial implementation of the binary. It’s expected to undergo significant changes and optimizations as I add missing features (e.g. indexer support).

Given this, I’d appreciate it if reviewers could focus more on the overall approach rather than the implementation details at this stage. Many of the current inefficiencies will likely go away as we continue to work on the code.